job-server/src/main/scala/spark/jobserver/io/FileCacher.scala

```diff
@@ -50,10 +50,10 @@ trait FileCacher {
       tempOutFileName, targetFullBinaryName: Any)
 
     val tempFile = new File(rootDir, tempOutFileName)
-    if( ! tempFile.renameTo(new File(rootDir, targetFullBinaryName))) {
+    if(!tempFile.renameTo(new File(rootDir, targetFullBinaryName))) {
```

Comment:
Sorry to be nitpicky, but could you please add a space between if and ( ?  Thanks